lib/p5.sound.js

```diff
@@ -1577,6 +1577,14 @@ var __WEBPACK_AMD_DEFINE_RESULT__;
 
 
   p5.Oscillator.prototype.fade = p5.Oscillator.prototype.amp;
+  /**
+   * Returns the value of output gain 
+   *
+   *  @method  getAmp
+   *  @for p5.Oscillator
+   * 
+   * @return  {number} Amplitude value between 0.0 and 1.0
```

Comment:
the tag we need is `@returns` with an "s" at the end

Reply:
i think making change in src/oscillator.js is enough , after that this will be build by grunt ?